Dockerfile.cloudbuild-agent

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:22.04
```

Comment:
We need to set up some kind of daily automated build for this. Can you add a TODO?